### ISSUE TITLE
NbSelect Change Detection Issue

### ISSUE THREAD

**Issue by dpollastrini** [akveo/nebular#1157]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [* ] bug report
* [ ] feature request

### Issue description

**Current behavior:**
When assigning [selected] input on NbSelect component in template, ExpressionChangedAfterItHasBeenCheckedError is thrown.

**Expected behavior:**
ExpressionChangedAfterItHasBeenCheckedError is not thrown.

**Steps to reproduce:**

Place nb-select in template with [selected] input set (see referenced StackBlitz) and load page. Observe error in console.

**Related code:**

[StackBlitz](https://angular-4yju13.stackblitz.io/)

### Other information:

**npm, node, OS, Browser**
```
> node --version
v10.15.0
> npm --version
6.4.1
```

**Angular, Nebular**
```
Node: 10.15.0
OS: win32 x64
Angular: 7.2.0
... animations, common, compiler, compiler-cli, core, forms
... http, platform-browser, platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.10.2
@angular-devkit/build-angular     0.10.2
@angular-devkit/build-optimizer   0.10.2
@angular-devkit/build-webpack     0.10.2
@angular-devkit/core              7.0.2
@angular-devkit/schematics        7.2.1
@angular/cdk                      7.2.1
@angular/cli                      7.2.1
@angular/language-service         7.0.0
@ngtools/webpack                  7.0.2
@schematics/angular               7.2.1
@schematics/update                0.12.1
rxjs                              6.3.0
typescript                        3.1.3
webpack                           4.19.1


    "@nebular/theme": "3.0.0",
```

**Comment by Eve-1995** [comment]
This problem performs in my project.

**Comment by kgrigorian** [comment]
having the same issue

**Comment by yggg** [comment]
Fix is ready (#1299) and it'll land in 3.4.1.
Before the release, you could prevent this error by triggering change detection manually in the component where select is used.
```
ngAfterViewInit() {
  // ...
  this.changeDetectorRef.detectChanges();
}
```

**Comment by kevinjamesbaker** [comment]
is this still an open issue?  I see that it is 'done' but I am seeing this interaction as well.

ERROR Error: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value: 'selected: false'. Current value: 'selected: true'.
